src/lcf.sig

```diff
@@ -1,19 +1,43 @@
+(* This signature should by the user of the logic. It requires them
+ * to specify the structure of targets of the tactics (what they should
+ * operate on, as well as what counts as evidence for having proven one.
```

Comment:
- This signature should by the user of the logic => This signature must be implemented for a logic in order to deploy the standard LCF tactical machinery. (or something like that).

- It requires them to specify the structure of the tactics... => An implementation of this signature includes specifications of goals, and what counts as the evidence of a goal.